packages/poseidon-proof/README.md

```diff
@@ -74,57 +74,55 @@ yarn add @zk-kit/poseidon-proof
 
 ## 📜 Usage
 
-\# **generate**(
-message: _BigNumberish_,
-scope: _BigNumberish_,
-snarkArtifacts?: _SnarkArtifacts_
-): Promise\<_PoseidonProof_>
-
-```typescript
-import { generate } from "@zk-kit/poseidon-proof"
-
-const scope = 1
-const message = 2
-
-const fullProof = await generate(message, scope)
-
-// If not specified, the Snark artifacts are downloaded automatically.
-// You can also specify them.
-const fullProof2 = await generate(message, scope, {
-    zkeyFilePath: "./poseidon-proof.zkey",
-    wasmFilePath: "./poseidon-proof.wasm"
-})
-
-console.log(fullProof)
-/*
-{
-    scope: '1',
-    hash: '8645981980787649023086883978738420856660271013038108762834452721572614684349',
-    nullifier: '7853200120776062878684798364095072458815029376092732009249414926327459813530',
-    proof: [
-        '8439157877715136449451128027974582198942685548176158240742469858289217640295',
-        '7742778415084033741154475792823260193410560424636243716622958831215275441500',
-        '19005176609208302398813682947612978224483117018101271134113308439452561466691',
-        '6209272538382784759793219866517186573065390524051102537220446165983489601194',
-        '12222537243697573476419214640884921904066033266502734603198915705889607365883',
-        '20066191345466355816238393590466176790809335456890624395337970890893401456064',
-        '4851746896803117511000878727783002155680855098198108089136166702412365578625',
-        '18391775232946751568173446051923718267369779168471637305003186214102154164036'
-    ]
-}
-*/
-```
-
-\# **verify**(poseidonProof: _PoseidonProof_): Promise\<_boolean_>
-
 ```typescript
-import { verify } from "@zk-kit/poseidon-proof"
-
-const response = await verify(fullProof)
-
-console.log(response) // true or false
-
-// Eventually you may want to check the nullifier.
+import { generate, verify } from "@zk-kit/poseidon-proof"
+;(async () => {
```

Comment:
This is no longer necessary with the new NodeJS versions (same comment for the other packages).

`process.exit(0)` could also be omitted imo.